src/Core/Checkout/Promotion/Subscriber/PromotionIndividualCodeRedeemer.php

```diff
@@ -64,7 +69,7 @@ public function onOrderPlaced(CheckoutOrderPlacedEvent $event): void
             // if we did not use an individual code we might have
             // just used a global one or anything else, so just quit in this case.
             if (!($individualCode instanceof PromotionIndividualCodeEntity)) {
-                return;
```

Comment:
Could you add a small unit test for this case? So we don't miss this anymore in the future.